charts/stable/leaf2mqtt/Chart.yaml

```diff
@@ -21,4 +21,4 @@ maintainers:
 dependencies:
 - name: common
   repository: https://library-charts.k8s-at-home.com
-  version: 2.5.0
+  version: 3.1.0
```

Comment:
```suggestion
  version: 3.1.1
```

Reply:
Ah so we prefer to follow the common chart version where possible? That's fine

Reply:
Ah - ignore this going crazy :)